scripts/utils/price_utils.js

```diff
@@ -201,7 +155,7 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
       return true
     }
 
-    const marketPrice = getUnitPrice(
+    const marketPrice = exchangeUtils.getUnitPrice(
```

Comment:
nit: The `getUnitPrice` should probably be imported directly from the contracts module:
```
const { getUnitPrice } = require("@gnosis.pm/dex-contracts")
```

And then:
```suggestion
    const marketPrice = getUnitPrice(
```